api/SpeechSynthesisUtterance.json

```diff
@@ -131,7 +131,9 @@
               "version_added": "33"
             },
             "chrome_android": {
-              "version_added": "33"
+              "version_added": "33",
+              "partial_implementation": true,
+              "notes": "Doesn't fire the <code>boundary</code> event as expected. See <a href='https://crbug.com/1122143'>bug 1122143</a>."
```

Comment:
I don't understand this note. What does it mean for the `boundary` event to not fire the `boundary` event?

Reply:
> I don't understand this note. What does it mean for the `boundary` event to not fire the `boundary` event?

The engine has support for the event, but due to some yet-unresolved bug in the implementation, it doesn’t fire when expected on mobile — somewhat similar to the case for https://github.com/mdn/browser-compat-data/pull/6890 (the `visibilitychange` event in Safari).

Reply:
Ah, OK. I misunderstood this in the first place—something about it struck me as recursive. What do you think of rephrasing this slightly?

```suggestion
              "notes": "The <code>boundary</code> event does not fire as expected. See <a href='https://crbug.com/1122143'>bug 1122143</a>."
```